Libraries/LibWebView/HeadlessWebView.h

```diff
@@ -35,6 +35,9 @@ class HeadlessWebView : public WebView::ViewImplementation {
     Web::Page::PendingDialog m_pending_dialog { Web::Page::PendingDialog::None };
     Optional<String> m_pending_prompt_text;
 
+    // When restoring from fullscreen, we need to know to what dimension.
+    Web::DevicePixelRect m_cached_dimensions;
```

Comment:
Calling this `m_cached_dimensions` seems to hint at some form of performance optimization. Maybe `m_previous_dimensions` is a better fit?

Reply:
A lot better, actually! Will change.